h2o-py/h2o/grid/infogram_grid.py

```diff
@@ -0,0 +1,44 @@
+import h2o
+
+
+def _train_and_get_models(model_class, x, y, train, **kwargs):
+    from h2o.automl import H2OAutoML
+    from h2o.grid import H2OGridSearch
+
+    model = model_class(**kwargs)
+    model.train(x, y, train)
+    if model_class is H2OAutoML:
+        return [h2o.get_model(m[0]) for m in model.leaderboard["model_id"].as_data_frame(False, False)]
+    elif model_class is H2OGridSearch:
+        return [h2o.get_model(m) for m in model.model_ids]
+    else:
+        return [model]
+
+
+def infogram_grid(infogram, model_class, y, training_frame, **kwargs):
```

Comment:
Can we add an example in the docstring?  I realize it will be a bit long... but maybe it would be helpful?